### PR TITLE
chore: add aot size analysis tool

### PR DESCRIPTION
**Pull request by Shanks0224**

### CODE REVIEW

File: test-tools/aot-analyzer/include/analyzer-error.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#ifndef ANALYZER_ERROR_H_
```
Comment (xujuntwt95329) [inline test-tools/aot-analyzer/include/analyzer-error.h:6]: Should add license header.

File: test-tools/aot-analyzer/src/aot-reader.cc [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  aot_get_module_mem_consumption((AOTModule *)module_, &mem_conspn_);
+  return Result::Ok;
+}
+
+Result AoTReader::ParseTargetInfoSection() {
+  uint32_t section_id = 0;
+  CHECK_RESULT(ReadU32(&section_id, "section_id"));
+  ERROR_UNLESS(section_id == AOT_SECTION_TYPE_TARGET_INFO, "bad section id");
+  uint32_t section_size = 0;
+  CHECK_RESULT(ReadU32(&section_size, "section_size"));
```
Comment (xujuntwt95329) [inline test-tools/aot-analyzer/src/aot-reader.cc]: Since we already load informations from `aot_module`, do we still need to read the raw bytes?

**Review by xujuntwt95329 — COMMENTED** [review]
Had better add a README file about the usage.

File: test-tools/aot-analyzer/src/aot-reader.cc (hunk 2 above)
Comment (Shanks0224) [reply test-tools/aot-analyzer/src/aot-reader.cc]: It is indeed somewhat repetitive to read raw bytes again here.
The `AoTModule` structure does not contain the `targetInfo`, and I cannot obtain the `targetInfo` information of the AoT file using the existing wamr interface.

File: test-tools/aot-analyzer/include/analyzer-error.h (hunk 1 above)
Comment (Shanks0224) [reply test-tools/aot-analyzer/include/analyzer-error.h:6]: done

**Pull request merged**